client/src/seenBy.tsx

```diff
@@ -15,25 +22,52 @@ export const SeenBy = ({ seenBy, userLookup }: SeenByProps) => (
       display: flex;
       align-items: center;
       justify-content: flex-end;
-      font-size: 80%;
     `}
   >
     <span
       css={css`
-        color: gray;
+        color: ${palette.neutral[46]};
         margin-right: 5px;
+        ${agateSans.xxsmall({ lineHeight: "tight" })}
       `}
     >
       Seen by
     </span>
-    {seenBy.map(({ userEmail, seenAt }) => (
-      <AvatarRoundel
+    {seenBy.slice(0, maxSeenByIcons).map(({ userEmail, seenAt }, i) => (
+      <div
         key={userEmail}
-        maybeUser={userLookup?.[userEmail]}
-        size={15}
-        userEmail={userEmail}
-        tooltipSuffix={` ${formattedDateTime(seenAt * 1000)}`}
-      />
+        css={css`
+          transform: translateX(-${i * roundelOverlapPct}%);
+          z-index: ${99999 - i};
+          height: ${roundelHeightPx}px;
+        `}
+      >
+        <AvatarRoundel
+          maybeUser={userLookup?.[userEmail]}
+          size={roundelHeightPx}
+          userEmail={userEmail}
+          tooltipSuffix={` ${formattedDateTime(seenAt * 1000)}`}
+        />
+      </div>
     ))}
+    {seenBy.length > maxSeenByIcons && (
+      <div
+        css={css`
+          transform: translateX(-${maxSeenByIcons * roundelOverlapPct}%);
+        `}
+      >
+        <span
```

Comment:
perhaps we could have a `title` tooltip, showing the email addresses of all the `seenBy`s that didn't make the cut

Reply:
great idea, done!